js/app.jsx

```diff
@@ -486,6 +486,26 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      var lines = [];
+      if (str) {
+        lines = str.split(/,|ÿ/)
+                   .map(val => { return val.trim(); })
+                   .map(val => {
+                    var matches = val.match(/\w*\.\w+\(\w+\.scala:\d+\)/i);
```

Comment:
I thought some context might be more meaningful than just a line number. So if the original line were:
`com.stripe.zoolander.parquet.ParquetJob$TypedPipeExtensions.save(ParquetJob.scala:201)`
we'd show:
`TypedPipeExtensions.save(ParquetJob.scala:201)`
The `\w*` at the front is for lines like `$.apply(...)`